packages/@guardian/source-react-components/src/icons/SvgCross.tsx

```diff
@@ -1,17 +1,36 @@
+import { css } from '@emotion/react';
 import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
-import { iconSize } from '@guardian/source-foundations';
+import { iconSize, visuallyHidden } from '@guardian/source-foundations';
 import type { IconProps } from './types';
 
-export const SvgCross = ({ size }: IconProps): EmotionJSX.Element => (
-	<svg
-		viewBox="0 0 30 30"
-		xmlns="http://www.w3.org/2000/svg"
-		width={size ? iconSize[size] : undefined}
-	>
-		<path
-			fillRule="evenodd"
-			clipRule="evenodd"
-			d="M15.325 17.025l7.225 6.625 1.075-1.075-6.6-7.25 6.6-7.25L22.55 7l-7.225 6.625-7.25-6.6L7 8.1l6.625 7.225L7 22.55l1.075 1.075 7.25-6.6z"
-		/>
-	</svg>
+export const SvgCross = ({
+	size,
+	isAnnouncedByScreenReader = false,
+}: IconProps): EmotionJSX.Element => (
+	<>
+		<svg
+			viewBox="0 0 30 30"
+			xmlns="http://www.w3.org/2000/svg"
+			width={size ? iconSize[size] : undefined}
+			aria-hidden={true}
+			focusable={false}
+		>
```

Comment:
I wonder if it's possible / helpful to abstract away this repetition in every `<svg>` element?

```tsx
<>
  <IconSvg size={size}>
    <path> ... </path>
  <IconSvg>
  { 
    // label code ...
  }
</>
```

Reply:
@mxdvl do you have any thoughts on this from an automation perspective? My thinking is:

- `<path>` elements are the unique part of an SVG. The `<svg>` element is likely to be identical for every icon
- if we optimise each SVG via SVGO or similar, the most important part of the transformation is likely to be in the `<path>` elements. If SVGO optimises the `<svg>` element, we are likely going to apply that to all SVGs, not just one or two